_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -7558,11 +7558,11 @@
 /obj/effect/mapping_helpers/airlock/unres{
 	dir = 4
 	},
-/obj/effect/mapping_helpers/airlock/access/all/engineering/maintenance,
-/obj/effect/mapping_helpers/airlock/access/all/service/kitchen,
 /obj/modular_map_root/tramstation{
 	key = "maintenance_uppermedsci"
 	},
+/obj/effect/mapping_helpers/airlock/access/any/engineering/maintenance,
+/obj/effect/mapping_helpers/airlock/access/any/medical/morgue,
```

Comment:
```suggestion
/obj/effect/mapping_helpers/airlock/access/any/medical/morgue,
```
Departmental maintenance should not allow people with just maint access. The convention is for departmental maintenance doors to have the departmental access only.

Reply:
Thank you, I will remember this in the future. Looks like this patch of maint had more problems than I thought it had.